app/code/core/Mage/Cms/Helper/Data.php

```diff
@@ -79,4 +79,152 @@ public function canUseCanonicalTag($store = null): bool
     {
         return Mage::getStoreConfigFlag(self::XML_PATH_USE_CMS_CANONICAL_TAG, $store);
     }
+
+    /**
+     * Returns an array of usages for a CMS page identifier in critical config (Home, No Route, No Cookies)
+     * Each entry describes where the page is used (Default Config, Website, Store View), ordered like in admin config.
+     *
+     * @param string $identifier
+     * @return array
+     */
+    public function getUsageScopes($identifier)
```

Comment:
```suggestion
    /**
     * Returns an array of usages for a CMS page identifier in critical config (Home, No Route, No Cookies)
     * Each entry describes where the page is used (Default Config, Website, Store View), ordered like in admin config.
     */
    public function getUsageScopes(string $identifier): array
```

Reply:
I have checked it and I can say that it is not implemented similarly. The extension is not necessary, since the solution works. The approval of this PR will make it mandatory to evaluate the remaining methods that are no longer needed, nor the local phrases.

Adding type hints and return types for sure improves code clarity, helps prevent type-related bugs, and aligns with modern PHP standards, making the code easier to maintain and analyze. However, in platforms like OpenMage/Magento 1, introducing strict typing may cause compatibility issues with older extensions or overrides that expect looser type requirements, so this change needs to be carefully considered and tested. Personally, I would prefer not to implement it for the moment.

Reply:
> However, in platforms like OpenMage/Magento 1, introducing strict typing may cause compatibility issues with older extensions or overrides that expect looser type requirements, so this change needs to be carefully considered and tested. Personally, I would prefer not to implement it for the moment.

Thanks for AI answer :P

Adding type hints and return types to new methods cant break anything. We did it for all PRs since months.